Documentation/PlatformSupport.md

```diff
@@ -0,0 +1,69 @@
+
+# Platform Support
+
+Darwin platforms are binary and source stable. Linux platforms follow semantic versioning. Windows follows semantic versioning.
+
+### Windows concerns
+
+Trying to perfectly unify Windows with Darwin and Linux at the API level is ultimately a futile endeavor. However, in some specific cases, Windows does have directly corresponding syscalls or near-equivalents.
+
+Windows-specific details are open to reevaluation pending more Windows expertise for the project.
+
+
+## API to syscall mapping
+
+### `Errno`
+
+#### Instance computed properties
+
+| API                | Darwin     | Linux      | Windows    |
+|:-------------------|:-----------|:-----------|:-----------|
+| `description`      | `strerror` | `strerror` | `strerror` |
+| `debugDescription` | `strerror` | `strerror` | `strerror` |
+
+
+### `FileDescriptor`
+
+#### Instance methods
+
+| API                          | Darwin      | Linux       | Windows                                            |
+|:-----------------------------|:------------|:------------|:---------------------------------------------------|
+| `open`                       | `open`      | `open`      | `_wsopen_s(..., _SH_DENYNO, _S_IREAD | _S_IWRITE)` |
+| `close`                      | `close`     | `close`     | `_close`                                           |
+| `seek`                       | `lseek`     | `lseek`     | `_lseeki64`                                        |
+| `read`                       | `read`      | `read`      | `_read`                                            |
+| `read(fromAbsoluteOffset:)`  | `pread`     | `pread`     | *custom*                                           |
+| `write`                      | `write`     | `write`     | `_write`                                           |
+| `write(fromAbsoluteOffset:)` | `pwrite`    | `pwrite`    | *custom*                                           |
+| `duplicate`                  | `dup`       | `dup`       | `_dup`                                             |
+| `duplicate(as:)`             | `dup2`      | `dup2`      | `_dup2`                                            |
+| `resize`                     | `ftruncate` | `ftruncate` | *N/A*                                              |
+
+
+#### Static methods
+
+| API    | Darwin | Linux  | Windows |
+|:-------|:-------|:-------|:--------|
+| `pipe` | `pipe` | `pipe` | *N/A*   |
```

Comment:
Yeah, sadly, my requests that the Windows side be added at the same time were ignored :-(.

Reply:
The Windows SDK is generally different enough that I think it makes sense for it to have its own code base, mostly/entirely separate from the rest of this package. I don't think one can convincingly argue that `CreatePipe` is close enough to `pipe(2)` that they need to have precisely the same API in swift-system, only differing on the backend -- can we?